documents.py

```diff
@@ -2,27 +2,22 @@
 
 
 # make gettext tools pick up these strings as translations
-def _(message):
-    return message
 
 
 class Documents(enum.Enum):
     HEADING_FOR_EXTINCTION_1 = {
-        "description": _("Why does XR exist? (the crisis)"),
+        "description": "Why does XR exist? (the crisis)",
         "filename": "heading_for_extinction_1.pdf",
     }
     HEADING_FOR_EXTINCTION_2 = {
-        "description": _("What does XR do? (the solution)"),
+        "description": "What does XR do? (the solution)",
         "filename": "heading_for_extinction_2.pdf",
     }
     XR_STRUCTURES = {
-        "description": _("How is XR organized? (the logistics)"),
+        "description": "How is XR organized? (the logistics)",
         "filename": "xr_structures.pdf",
     }
     XR_UNOFFICIAL_GUIDE = {
-        "description": _("But really, how does this work? (the reality)"),
+        "description": "But really, how does this work? (the reality)",
         "filename": "xr_unofficial_guide.pdf",
     }
-
-
-del _
```

Comment:
Due to the removal of the `_` (or, in the new situation, `translate`) function here, we cannot automatically extract strings for translations from this file anymore (using tools like gettext). 
I got the idea for these 'deferred translations'  from here: https://docs.python.org/3/library/gettext.html#deferred-translations
Could you please have a look and see if we can restore this functionality?

Reply:
This has been fixed in #9